scss/components/_card.scss

```diff
@@ -56,6 +56,7 @@ $card-margin: $global-margin !default;
   @if $global-flexbox {
     display: flex;
     flex-direction: column;
+    flex-grow: 1;
```

Comment:
I'm confused about this... Why are we pre-assuming how a card should behave relative to its parent? AFAIK we don't have any knowledge of whether the card will be a flex child or not